app/views/patients/update.js.erb

```diff
@@ -27,6 +27,7 @@
     $('#sections').append(
       '<%= escape_javascript(render partial: "patients/fulfillment", locals: { patient: @patient })%>'
     );
+    <% @patient.update_attributes urgent_flag: false %>
```

Comment:
The idea here is dead on, but can we put this into the patients controller? Specifically, I'd like to see this happen as part of the update action -- we don't want to change object state in our views if we can possibly avoid it.